plutus-playground-client/README.md

```diff
@@ -13,8 +13,7 @@ The Plutus Playground is written in [PureScript](https://www.purescript.org/) an
 $ plutus-playground-server
 ```
 
-The `plutus-playground-server` script is provided by the global [shell.nix](../shell.nix) and starts the server (If the command
-is not available make sure you are in a nix-shell session or that lorri is ready). For additional information on invoking the backend server please refer to its [README.md](https://github.com/input-output-hk/plutus-apps/tree/main/plutus-playground-server).
+The `plutus-playground-server` script is provided by the global [shell.nix](../shell.nix) and starts the server (If the command is not available make sure you are in a nix-shell session or that lorri is ready). For additional information on invoking the backend server please refer to its [README.md](https://github.com/input-output-hk/plutus-apps/tree/main/plutus-playground-server).
```

Comment:
(If the command is not available) -> (if the comment is not available,)

For additional information on invoking the backend server -> add comma